internal_test_data/minimal_test_case.yaml

```diff
@@ -16,3 +16,5 @@ stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_profile_4mv_5queries.y
 db_type: scylla
 instance_type_db: 'i3.large'
 ami_id_db_scylla: 'ami-06f919eb'
+
+nemesis_include_filter: [['topology_changes'],['schema_changes']]
```

Comment:
can we rename this configuration to `nemesis_selector` or something similar, include and filter are kind of the opposites, and it's a confusing name.

Reply:
We can, I didn't have any other good name.
But this PR isn't the place.

Reply:
I've seen this PR adding this to `docs/configuration_options.md` so I assumed it's a new configuration option this PR introduced. (didn't see it being used till now)

Now I see it's already in place.